app/models/story.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+##
+# Primary container for the data associated with a contributed story.
+class Story
+  include Mongoid::Document
+  include Mongoid::Timestamps
+
+  belongs_to :ore_aggregation, class_name: 'ORE::Aggregation', inverse_of: :story,
```

Comment:
should this relationship be flipped? As in the aggregation belongs to the story?

Reply:
is it possible to do a has_one/has_one one to one relationship in mongo?

Reply:
> should this relationship be flipped? As in the aggregation belongs to the story?

I don't think so. The reference is on the story, to the aggregation.

> is it possible to do a has_one/has_one one to one relationship in mongo?

Do you mean has_one/belongs_to? If so, yes, that's what we have here. If you meant what your wrote, i.e. has_one/has_one, what does that mean?

Reply:
right, I guess my thinking was that potentially it's more likely to have a story with multiple aggregations as opposed to an aggregation with multiple stories. Realistically though this relationship is intended to always be one to one. 

Regarding the has_one/has_one, yeah I was thinking the way this was persisted would result in a nested document in mongo. The has_one/belongs_to is in fact fine if we can assume that it is always going to stay as a one to one relationship.

Reply:
Yes, it will always be one-to-one, and one of the incentives for creating the distinct `Story` model is to avoid pollution of the EDM class documents like ORE::Aggregation, by which reasoning it would be preferable not to link it to an instance of the domain-specific Story class.